admin/src/main/ts/src/app/management/message-flash/form/message-flash-preview/message-flash-preview.component.html

```diff
@@ -3,9 +3,17 @@ <h3><s5l>flashmsg.preview</s5l> :</h3>
   <div class="twelve cell">
     <div *ngIf="customColor != undefined"  class="flashmsg" [ngStyle]="{'background-color': customColor}">
       <div [innerHTML]="domSanitizer.bypassSecurityTrustHtml(text)"></div>
+      <div *ngIf="!!signature" style="font-style: italic;" [ngStyle]="{'color': signatureColor}">
```

Comment:
éviter le style dans l'attribut style au maximum, à la place écrire une classe 'is-italic' dans admin.scss, comme pour le bold, exemple : https://github.com/opendigitaleducation/entcore/blob/5432bbd33d3fdae0305d833113582237e3f03a37/admin/src/main/ts/src/assets/admin.scss#L689